opentracing-shim/src/test/java/io/opentelemetry/opentracingshim/TracerShimTest.java

```diff
@@ -135,6 +135,9 @@ void activateSpan_withBaggage() {
 
       assertThat(io.opentelemetry.api.trace.Span.current()).isSameAs(actualSpan);
       assertThat(io.opentelemetry.api.baggage.Baggage.current()).isSameAs(actualBaggage);
+
+      Span child = tracerShim.buildSpan("child").start();
+      assertThat(child.getBaggageItem("foo")).isNotNull();
```

Comment:
it should be equal to `bar` shouldn't it?

Reply:
right - fixed